src/ShadowDOM/wrappers.js

```diff
@@ -182,7 +182,7 @@ window.ShadowDOMPolyfill = {};
 
   function getDescriptor(source, name) {
     try {
-      return Object.getOwnPropertyDescriptor(source, name);
+      return Object.getOwnPropertyDescriptor(source, name) || dummyDescriptor;
```

Comment:
My assumption was that showModalDialog could be the first of many such errors. Turns out it's the only one. I've updated the PR.